tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/legalize_to_linalg.cc

```diff
@@ -1729,6 +1761,27 @@ struct NormalConvOpConversion : public OpConversionPattern<mhlo::ConvOp> {
     // TODO(ataei): Only support dilated kernel right now. We need to consider
     // input dilation for deconvolution cases.
     Attribute dilations = op.rhs_dilationAttr();
+
+    // Check if padding is zero.
```

Comment:
I think you should improve the comment here, it should be a few lines to describe what's happening here, possibly explaining how the padding is handled on the input.